deploy/helm/kubecf/assets/operations/instance_groups/eirini-helm.yaml

```
@@ -1,22 +1,17 @@
 {{- if  .Values.features.eirini.enabled }}
+
+  {{- /* *** Make sure the eirini stack has been selected *** */}}
+  {{- $stack := include "_config.lookup" (list $ "features.eirini.stack") }}
+  {{- $ok := eq (len $.Values.install_stacks) 1 }}
+  {{- $ok = and $ok (eq $stack (first $.Values.install_stacks)) }}
```

I think in this particular case using pipes is easier to read:
```suggestion
  {{- $ok := len $.Values.install_stacks | eq 1 }}
  {{- $ok = first $.Values.install_stacks | eq $stack | and $ok }}
```
(Because they are commutative, this ends up reading kind of English-y.)

---

Change applied in 0ab9ace